app/views/tag/_advanced_tagging.html.erb

```diff
@@ -29,7 +29,7 @@
 
   $(document).ready(function(){
 
-    $( "#pt-list > li " ).each(function() {
```

Comment:
ah cool, i see!

OK so i think you can actually add the reload within the function below then? to avoid the href usage... does that work? Thanks!